GDAXSharp/Services/Orders/OrdersService.cs

```diff
@@ -27,14 +24,16 @@ public class OrdersService : AbstractService
         public async Task<OrderResponse> PlaceMarketOrderAsync(
             OrderSide side,
             ProductType productId,
-            decimal size)
+            decimal size,
+            Guid clientOid = new Guid())
         {
             var order = new Order
             {
                 Side = side,
                 ProductId = productId,
                 OrderType = OrderType.Market,
-                Size = size
+                Size = size,
+                ClientOid = clientOid
```

Comment:
if the user decides not to pass in a `Guid` wouldn't this always pass in a `Guid` of `00000000-0000-0000-0000-000000000000`?

Reply:
Oops, fixed that. Now it's nullable 😄

Reply:
@Sotam i think this is okay now. is it a problem passing in null for clientId though? i see orders can still be placed... not sure how gdax actually handles it.